qa/tasks/cephfs/caps_helper.py

```diff
@@ -128,7 +166,16 @@ class CapTester(CephFSTestCase):
     run_mon_cap_tests() or run_mds_cap_tests() as per the need.
     """
 
-    def write_test_files(self, mounts, testpath=''):
+    def __init__(self, mount=None, path='', create_test_files=True):
```

Comment:
Are there any callers with `create_test_files` unset in this change?

Reply:
I planned to use it in a couple of places where `create_test_files` would be unset but there's no such instance yet....

Reply:
I've deleted the commit that contained this code.

Reply:
Then it that case it very much looks like this change (if not all) should be a part of the other PR.

My point is - this change aims to "improve caps helpers", so much of the improvement should be specific to this change. Its not incorrect to introduce changes that do not make sense (directly) but when one has to go through a range of commits to infer what and when changes were introduced, its a bit hard to join the dots at that point.